plugins/pvcmount.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: pvcmount
+spec:
+  version: "v1.0.0"
+  shortDescription: Go binary for mounting persistent volumes.
```

Comment:
Nobody cares if it's a go binary :) 

Lets explain the functionality here please, such as:

> Mount PersistentVolumes to Pods for debugging